src/coverlet.collector/coverlet.collector.csproj

```diff
@@ -15,6 +15,7 @@
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);PackBuildOutputs</TargetsForTfmSpecificContentInPackage>
     <!-- Open issue https://github.com/NuGet/Home/issues/8941 -->
     <NoWarn>$(NoWarn);NU5127</NoWarn>
+    <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
```

Comment:
In the PR summary you wrote: 
> disable `CentralPackageTransitivePinningEnabled` for projects generating nuget packages

Is this then also needed for `coverlet.console`? Because it also ships as nuget package as dotnet tool?

Reply:
The transitive dependencies updated should not happen whenever the compatibility to vstest is required. coverlet.console is a standalone dotnet tool and not run by test host.
image.png